src/main/java/com/adelinasportfolio/emailservice/model/EmailEntity.java

```diff
@@ -0,0 +1,19 @@
+package com.adelinasportfolio.emailservice.model;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class EmailEntity {
+    String to;
```

Comment:
Poate fi mai bine "recipient", sa fie mai clar?

LE: Dar totusi clasa e EmailEntity, asa ca e destul de clar pentru ce e "to".

Reply:
Am pus tu pentru ca și SpringEmail folosește to. Sa fie mai clar.